MediaPlayer/MediaControls.js

```diff
@@ -225,7 +237,7 @@ const MediaControls = kind({
 				<Button
 					aria-label={$L('Shuffle')}
 					backgroundOpacity="transparent"
-					className={shuffle ? css.repeat : ''}
+					className={shuffleButtonClassName}
```

Comment:
I think this can be achieved in a simpler way.
The text-color of the Button which has the `repeat` className can be customized.
The only skin which seems to have a problem is Cobalt. For copper, with the latest develop branch code, it looks like this , which looks OK

![image](https://user-images.githubusercontent.com/63335068/117928172-d719d400-b303-11eb-8e87-a4b223b4cad1.png)

Reply:
Solved